src/main/resources/templates/fragments/navbar.html

```diff
@@ -0,0 +1,32 @@
+<header th:fragment="navbar" id="navbar" aria-label="Main navigation">
```

Comment:
create namespace declaration to remove the red warning from th

Reply:
done